src/components/articles/AllArticles.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../utils/getApi";
+import ArticleItem from "./ArticleItem";
+
+export default function AllArticles() {
+  const [allArticles, setAllArticles] = useState([]);
+  useEffect(() => {
+    getArticles().then(({ articles }) => {
+      setAllArticles(articles);
+    });
```

Comment:
Good work setting up an extracted, reusable function here.

I can see you implemented an `isLoading` pattern in your other PR: [https://github.com/keaysb/nc-news/pull/3/files#diff-346d21e16fe5dd73c4e4bce9222b8dcd7bf6b589d82be8cae3a0638205f1419d](url)